### PR TITLE
project-wizard updated: validator setting implemented, style problems…

### PR DESCRIPTION
**Pull request by alexAlice**

### CODE REVIEW

File: packages/donations-app/devServer/controller/project.js [hunk 1]
```diff
@@ -175,9 +176,20 @@ module.exports = function (app) {
...
           project: savedProject,
           outcomes: req.body.outcomes,
+          validators: req.body.validators,
           changedBy: req.user._id
         }).save();
 
+        // Setting validator
+        if (AccessControl.isSuperadmin(req.user)
+            && req.body.validators
+            && req.body.validators.length > 0)
```
Comment (jakub-wojciechowski) [inline packages/donations-app/devServer/controller/project.js]: shouldn't we enforce length == 1?

File: packages/donations-app/devServer/controller/project.js [hunk 2]
```diff
@@ -175,9 +176,20 @@ module.exports = function (app) {
         await new ProjectHistory({
           project: savedProject,
           outcomes: req.body.outcomes,
+          validators: req.body.validators,
           changedBy: req.user._id
         }).save();
 
+        // Setting validator
+        if (AccessControl.isSuperadmin(req.user)
```
Comment (jakub-wojciechowski) [inline packages/donations-app/devServer/controller/project.js]: It's sufficient to require that the user is just charity-admin, we plan to gradually open the wizard to the public.

File: packages/donations-app/devServer/controller/project.js [hunk 3]
```diff
@@ -175,9 +176,20 @@ module.exports = function (app) {
...
+            && req.body.validators
+            && req.body.validators.length > 0)
+        {
+          let user = await User.findById(req.body.validators[0]);
+          user.validator.push(savedProject._id);
```
Comment (jakub-wojciechowski) [inline packages/donations-app/devServer/controller/project.js]: Do we handle the update? 
Are we doing anything to the previous validator (like cleaning the project from the previous user validators list)?

File: packages/donations-app/devServer/controller/project.js (hunk 2 above)
Comment (alexAlice) [reply packages/donations-app/devServer/controller/project.js]: I thought that charity admin should not be able to set validator

**Pull request merged**